woo-components/src/components/Pagination/Pagination.jsx

```diff
@@ -0,0 +1,48 @@
+const React = require('react');
+const { PropTypes } = React;
+
+class Pagination extends React.Component {
+
+  render () {
+    const PageLinks = ({amount}) => {
+      return (
+        <div>
+          {
+            Array(amount).fill().map((_, i) => {
```

Comment:
This will also break if you have too many pages because there is not fixed limit set on how many max page buttons to render.